content/docs/2.6/scalers/azure-pipelines.md

```diff
@@ -15,8 +15,10 @@ This specification describes the `azure-pipelines` trigger for Azure Pipelines.
 triggers:
   - type: azure-pipelines
     metadata:
-      # Required - Learn more in 'How to determine your pool ID'
+      # Optional: Learn more in 'How to determine your pool ID'
       poolID: "{agentPoolId}"
+      # Optional
+      poolName: "{agentPoolName}"
```

Comment:
Should we clarify the difference? What is the pool name, is that the name from the UI? If so, let's clarify in the parameter list